jadx-gui/src/main/java/jadx/gui/device/debugger/BreakpointManager.java

```diff
@@ -166,7 +166,7 @@ protected String getFullMthRawID() {
 
 		@Override
 		public int hashCode() {
-			return (int) (31 * codeOffset + 31 * cls.hashCode() + 31 * mth.hashCode());
+			return 31 * (int) codeOffset + 31 * cls.hashCode() + 31 * mth.hashCode();
```

Comment:
This is quite bad implementation of `hashCode`. If you are not sure how to write it correctly just use buildin helper method:
```suggestion
			return Objects.hash(codeOffset, cls, mth);
```